main.go

```diff
@@ -46,6 +46,12 @@ func main() {
 		log.Fatal("js-path is required")
 	}
 
+	if glideYMLFile, err := os.Open("glide.yml"); err == nil {
```

Comment:
Oh interesting, I didn't think about going that far with this check. I'd be fine starting without enforcing glide, and later adding it in if we find it necessary.